package.json

```diff
@@ -30,10 +34,13 @@
     "validate": "office-addin-manifest validate manifest.xml",
     "watch": "webpack --mode development --watch"
   },
+  "dependencies": {
+    "core-js": "^3.9.1",
+    "regenerator-runtime": "^0.13.7"
+  },
   "devDependencies": {
-    "@babel/core": "^7.12.10",
-    "@babel/polyfill": "^7.12.1",
-    "@babel/preset-env": "^7.12.11",
+    "@babel/core": "^7.13.10",
+    "@babel/preset-typescript": "^7.13.0",
```

Comment:
should have dependency on  @babel/present-env  and remove @babel/preset-typescript

Reply:
Wasn't it your change that swapped @babel/present-env for @babel/preset-typescript?

Reply:
For the TypeScript projects yes. As I had commented before, the JS projects should use @babel/present-env.

Reply:
The original comment here still applies. Please update. Thanks.

Reply:
OK, the dependency is for the tests. Since the tests get removed in convert-to-single-host.js, perhaps it would make sense to remove the dependencies on the typescript stuff. It's not ideal, but I think it is OK for now. Could you open an issue to track doing cleanup of TS dependencies when convert-to-single-host.js is run in -JS projects?